### PR TITLE
Railjson v2.2.0: remove generated data

### LINKED ISSUE #612: Remove all generated data from RailJSON
This is needed to migrate to generated tables

- This define a new version of railjson (`2.1.0`). 
- Endpoints and commands to export / import railjson must be adapt.

### PR DESCRIPTION
**Pull request by flomonster**
close #612

### CODE REVIEW

**Review by multun — DISMISSED** [review]

File: core/src/main/java/fr/sncf/osrd/railjson/schema/infra/RJSInfra.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
 import fr.sncf.osrd.railjson.schema.infra.trackobjects.RJSBufferStop;
 import fr.sncf.osrd.railjson.schema.infra.trackobjects.RJSSignal;
 import fr.sncf.osrd.railjson.schema.infra.trackobjects.RJSTrainDetector;
+import fr.sncf.osrd.railjson.schema.infra.trackranges.RJSSpeedSection;
+
```
Comment (eckter) [inline core/src/main/java/fr/sncf/osrd/railjson/schema/infra/RJSInfra.java]:
```suggestion
```

checkstyle error

File: core/src/main/java/fr/sncf/osrd/railjson/parser/RailJSONParser.java [hunk 2]
```diff
@@ -233,16 +246,6 @@ private static TrackSection parseTrackSection(
         var infraTrackSection = trackGraph.makeTrackSection(beginID, endID, trackSection.id,
```
Comment (eckter) [inline core/src/main/java/fr/sncf/osrd/railjson/parser/RailJSONParser.java]:
```suggestion
        final var infraTrackSection = trackGraph.makeTrackSection(beginID, endID, trackSection.id,
```

checkstyle error

File: api/osrd_infra/schemas/infra.py [hunk 3]
```diff
@@ -1,14 +1,17 @@
 from enum import Enum
```
Comment (eckter) [inline api/osrd_infra/schemas/infra.py:1]: The infra generator script needs to be adapted to these changes

Comment (flomonster) [reply api/osrd_infra/schemas/infra.py:1]: Done

**Review by multun — APPROVED** [review]

**Pull request merged**